### ISSUE TITLE
azurerm_app_service does not include "outboundIpAddresses" in attributes or state file

### ISSUE THREAD

**Issue by TheLinuxNinja** [terraform-providers/terraform-provider-azurerm#360]
### Terraform Version
Terraform v0.10.6

### Affected Resource(s)
azurerm_app_service

### Terraform Configuration Files
```hcl
resource "random_id" "test-name-nonce" {
  byte_length = 8
}

resource "azurerm_app_service" "test-web-app" {
  name                = "test-web-app-${random_id.test-name-nonce.b64_url}-${var.environment}-${var.region}"
  location            = "${var.region_description}"
  resource_group_name = "${azurerm_resource_group.test-rg.name}"
  app_service_plan_id = "${azurerm_app_service_plan.test-service-plan.id}"

  site_config {
    always_on                = true
    dotnet_framework_version = "v4.0"

    default_documents = [
      "Default.htm",
      "Default.html",
      "Default.asp",
      "index.htm",
      "index.html",
      "iisstart.htm",
      "default.aspx",
      "index.php",
      "hostingstart.html",
    ]
  }

  app_settings {
    WEBSITE_NODE_DEFAULT_VERSION = "x.x.x"
  }

  client_affinity_enabled = true
  enabled                 = true

  tags {
    environment = "${var.environment}"
    region      = "${var.region}"
  }
}
```

### Debug Output
n/a

### Panic Output
n/a

### Expected Behavior
Creating a azurerm_app_service should also make the "outboundIpAddresses": available in attributes and in the state file.

### Actual Behavior
"outboundIpAddresses" is not in the state file and does not appear to be available as an attribute.

### Steps to Reproduce
create azurerm_app_service resource
examine the tfstate file

### Important Factoids
n/a

### References
As this is a new resource in Terraform, it is likely that there are many attributes and configuration options that have yet to be added. This is one of them. The use case would be for these "outboundIpAddresses" to be used in azurerm_network_security_rule resources or other resources managed by Terraform.

**Comment by horsey** [comment]
Using Terraform v0.11.1
+ provider.azurerm v1.1.0

**Comment by tombuildsstuff** [comment]
[Fixed in v1.1.0](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/CHANGELOG.md#110-january-26-2018)
